Documentation/observability/metrics.rst

```diff
@@ -530,6 +539,29 @@ and use the IP address of the target instead.
 
    In all of these 3 cases, ``reserved-identity`` context returns ``reserved:kube-apiserver``.
 
+Hubble metrics can also be configured with a ``labelsContext`` which allows providing a list of labels
+that should be added to the metric. Unlike ``sourceContext`` and ``destinationContext``, instead
+of different values being put into the same metric label, the ``labelsContext`` puts them into different label values.
+
+========================= ===============================================================================
+Option Value              Description
+========================= ===============================================================================
+``source_namespace``      The namespace of the pod if the flow source is from a Kubernetes pod.
+``source_pod``            The pod name if the flow source is from a Kubernetes pod.
+``source_workload``       The name of the source pod's workload (Deployment, Statefulset, Daemonset, ReplicationController, CronJob, Job, DeploymentConfig (OpenShift)).
+``source_app``            The app name of the source pod, derived from pod labels (``app.kubernetes.io/name``, ``k8s-app``, or ``app``).
+``destination_namespace`` The namespace of the pod if the flow destination is from a Kubernetes pod.
+``destination_pod``       The pod name if the flow destination is from a Kubernetes pod.
+``destination_workload``  The name of the destination pod's workload (Deployment, Statefulset, Daemonset, ReplicationController, CronJob, Job, DeploymentConfig (OpenShift)).
+``destination_app``       The app name of the source pod, derived from pod labels (``app.kubernetes.io/name``, ``k8s-app``, or ``app``).
+``reporter``              Identifies the origin of the flow/request. It is set to ``source`` if it originated from the client, and ``destination`` if it originated from the server.
```

Comment:
:+1: for source/destination. Also, maybe document `unknown` value?

Reply:
Ah, I originally used source/destination, but I switched to `client/server` which is inline with the OTEL semantics conventions (uses client/server terminology).

Reply:
I updated the documentation to reflect the usage of `client/server`. If you think we should use source/destination, we could also consider adding another label option such as `traffic_direction` which just exposes the value as `egress|ingress|unknown`, even though it overlaps with `reporter`. Might be more useful for non-HTTP flows where client/server might be less useful. 

Another option is to remove `reporter` from the contextOptions and to put it directly on the HTTP metrics labels themselves. I'm actually thinking this might be best.

Reply:
Ok so I ended up doing this:
- Replace `reporter` contextLabel option with `traffic_direction`, which has label values `ingress|egress|unknown`. 
- Add `reporter` label directly to HTTP metrics with values `client|server|unknown`. 

This is also great because it ensures the `reporter` label is present on HTTP metrics when no contextLabel options are configured, which means it's more likely that users pay attention to it when writing queries, which should help with preventing people from double counting when using egress/ingress policies on the same port.

Reply:
Having `reporter` label directly on metrics makes sense to me, but what's the value of a separate `traffic_direction` label then?

Reply:
Only if you enable it. It's two ways of saying the same thing, but in the context of `client|server` I think that doesn't always make sense for all flows, so you may use `reporter` for http, and `traffic_direction` for say: flow metrics.

Reply:
I see. I think as a user I would rather avoid duplicated information and cope with `client|server` for flows rather than enable `traffic_direction`. But no strong opinion on that.

Reply:
Information is only duplicated if you explicitly enable `traffic_direction`, I think it's not a huge deal here.